src/ConfigFile.c

```diff
@@ -862,6 +890,12 @@ void Ros_ConfigFile_Parse()
 
     Ros_ConfigFile_ValidateCriticalSettings();
     Ros_ConfigFile_ValidateNonCriticalSettings();
+#if defined(YRC1000) || defined(YRC1000u)
+    if(g_nodeConfigSettings.userlan_debug_broadcast_enabled &&
+        (g_nodeConfigSettings.userlan_debug_broadcast_port == CFG_ROS_USER_LAN1 ||
+        g_nodeConfigSettings.userlan_debug_broadcast_port == CFG_ROS_USER_LAN2))
+        Ros_Debug_SetFromConfig();
```

Comment:
```suggestion
    if(g_nodeConfigSettings.userlan_debug_broadcast_enabled &&
        (g_nodeConfigSettings.userlan_debug_broadcast_port == CFG_ROS_USER_LAN1 ||
        g_nodeConfigSettings.userlan_debug_broadcast_port == CFG_ROS_USER_LAN2))
    {
        Ros_Debug_SetFromConfig();
    }
```

please always enclose bodies of conditionals in brackets. Just to make my internal parser happy perhaps, but clearer in any case.